### PR TITLE
[full-ci][tests-only] Add internal link e2e tests for project space

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Added internal link e2e test journey for the project space

## Related Issue
Part of https://github.com/owncloud/web/issues/8101

## How Has This Been Tested?
local

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

File: tests/e2e/support/objects/app-files/page/public.ts [hunk 1]
```diff
@@ -72,6 +72,15 @@ export class Public {
     await this.#page.locator('body').click()
   }
 
+  async uploadInternal(args: Omit<uploadResourceArgs, 'page'> & { link: string }): Promise<void> {
+    // link is the public link url
+    const { link } = args
+    delete args.link
```
Comment (SwikritiT) [inline tests/e2e/support/objects/app-files/page/public.ts:78]: why delete this?

Comment (saw-jan) [reply tests/e2e/support/objects/app-files/page/public.ts:78]: link is not needed for the next function. So, extracted it and then deleted from the args list

File: tests/e2e/support/objects/app-files/page/public.ts [hunk 2]
```diff
@@ -72,6 +72,15 @@ export class Public {
...
+    await uploadResource({ ...args, page: this.#page })
+    await this.#page.goto(link)
+    await this.#page.locator('body').click()
```
Comment (ScharfViktor) [inline tests/e2e/support/objects/app-files/page/public.ts:81]: Do we need to take these steps? I think it's unnecessary

**Review by ScharfViktor — APPROVED** [review]
Other than what I wrote earlier everything else is cool, thanks

File: tests/e2e/support/objects/app-files/page/public.ts (hunk 2 above)
Comment (saw-jan) [reply tests/e2e/support/objects/app-files/page/public.ts:81]: was just a copy-paste. I will have a look. Thanks!

**Review by amrita-shrestha — COMMENTED** [review]
other than existing comment lgtm

**Pull request merged**